lib/adapters/autocomplete-adapter.ts

```diff
@@ -84,12 +92,25 @@ export default class AutocompleteAdapter {
 
     // Get the suggestions either from the cache or by calling the language server
     const suggestions = await
-      this.getOrBuildSuggestions(server, request, triggerChar, triggerOnly, onDidConvertCompletionItem);
+      this.getOrBuildSuggestions(server, request, triggerChar, triggerOnly, request.prefix, onDidConvertCompletionItem);
```

Comment:
It already gets `request`; does it need `request.prefix` separately?

Reply:
Ah good catch, totally missed that. I fixed that in a separate commit. Let me know if you prefer them squashed instead. Thanks!